engine/job.go

```diff
@@ -45,9 +39,10 @@ type Job struct {
 
 // Run executes the job and blocks until the job completes.
 // If the job fails it returns an error
-func (job *Job) Run() error {
+func (job *Job) Run() (err error) {
 	if job.Eng.IsShutdown() && !job.GetenvBool("overrideShutdown") {
-		return fmt.Errorf("engine is shutdown")
+		err = fmt.Errorf("engine is shutdown")
```

Comment:
Actually this makes no sense. Whole point of named return, that it will be assigned on return from function. like this you could use just `var err error`